components/Header.js

```diff
@@ -59,39 +58,39 @@ class Header extends React.Component {
     return title;
   }
 
+  getTwitterHandle() {
+    const { collective } = this.props;
+    const parentCollective = collective?.parentCollective;
+    const handle = this.props.twitterHandle || collective?.twitterHandle || get(parentCollective, 'twitterHandle');
+    return handle ? `@${handle}` : '';
+  }
+
   getMetas() {
     const { noRobots, collective } = this.props;
     const title = this.props.title || (collective && collective.name);
     const image = this.props.image || (collective && getCollectiveImage(collective));
-    const description =
-      this.props.description || (collective && (collective.description || collective.longDescription));
-    const twitterHandle =
-      this.props.twitterHandle ||
-      (collective && (collective.twitterHandle || get(collective.parentCollective, 'twitterHandle')));
-
+    const description = this.props.description || collective?.description || collective?.longDescription;
     const metaTitle = title ? `${title} - Open Collective` : 'Open Collective';
     const defaultImage = `https://opencollective.com/static/images/opencollective-og-default.png`;
 
-    const metas = {
-      'twitter:site': '@opencollect',
-      'twitter:creator': twitterHandle ? `@${twitterHandle}` : '',
-      'fb:app_id': '266835577107099',
-      'og:image': image || defaultImage,
-      description: truncate(description, 256),
-      'og:description': truncate(description, 256),
-      'twitter:card': 'summary_large_image',
-      'twitter:title': metaTitle,
-      'twitter:description': truncate(description, 256),
-      'twitter:image': image || defaultImage,
-      'og:title': metaTitle,
-      ...this.props.metas,
-    };
+    const metas = [
+      { property: 'twitter:site', content: '@opencollect' },
+      { property: 'twitter:creator', content: this.getTwitterHandle() },
+      { property: 'fb:app_id', content: '266835577107099' },
+      { property: 'og:image', content: image || defaultImage },
+      { property: 'og:description', name: 'description', content: truncate(description, 256) },
```

Comment:
As per https://www.geeksforgeeks.org/whats-the-difference-between-meta-name-and-meta-property/, `description` and `og:description` can be set with the same tag.